### PR TITLE
Blog post on securing a Spring microservices architecture

### PR DESCRIPTION
**Pull request by mraible**
## Description:

Blog post on securing a Spring microservices architecture with Spring Security, Juiser, Okta and the Sign-In Widget

### CODE REVIEW

File: _source/_posts/2017-08-08-secure-spring-microservices.md [hunk 1]
```diff
@@ -0,0 +1,650 @@
...
+To begin, you'll need to clone the aforementioned article's completed project.
+
+```bash
+git clone https://github.com/oktadeveloper/spring-boot-microservices-example.git
+```
+
+[Create an Okta Developer account](https://github.com/stormpath/stormpath-sdk-java/blob/okta/OktaGettingStarted.md). After completing these steps, you should have the information you need to set the following environment variables.
+
+```bash
+export STORMPATH_CLIENT_BASEURL=[base-url]
```
Comment (bretterer) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: Is this supposed to be `STORMPATH_CLIENT_BASEURL` or has it been changed to `OKTA`?

File: _source/_posts/2017-08-08-secure-spring-microservices.md [hunk 2]
```diff
@@ -0,0 +1,650 @@
...
+Then add a dependency for Stormpath's Zuul integration.
+
+```xml
+<dependency>
+    <groupId>com.stormpath.spring</groupId>
+    <artifactId>stormpath-zuul-spring-cloud-starter</artifactId>
+</dependency>
+```
+
+> **NOTE:** I'm using Stormpath libraries in this example. We plan to release Okta libraries that have this same functionality soon. I'll make sure to update this post when the Okta Zuul support has been released.
```
Comment (bretterer) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: Maybe move this above all the stormpath references.  I had thoughts while reading up to this point wondering if it was a typo that you kept everything as `stormpath`.  Didn't realize until just now that the library has not been updated to Okta

File: _source/_posts/2017-08-08-secure-spring-microservices.md (hunk 1 above)
Comment (mraible) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: Yep, unless @bdemers changed something recently.

File: _source/_posts/2017-08-08-secure-spring-microservices.md (hunk 2 above)
Comment (mraible) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: OK, the Stormpath library works with Okta, it just retains the Stormpath name.

Comment (bretterer) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: No problem.. could have just be me being a little confused on it.

**Comment by mraible** [comment]
@bretterer I moved the note up in https://github.com/okta/okta.github.io/pull/1140/commits/6ab8adc06ff711de7df064971ee62d807295ee0c.

**Comment by bretterer** [comment]
👍

**Pull request merged**